### PR TITLE
Feature: create diagram from ia

### PR DESCRIPTION
**Pull request by Zorin95670**

### CODE REVIEW

File: src/services/AIService.js [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+/**
+ * Generate diagrams using Leto AI proxy.
+ * It posts a request to the proxy and returns a json response composed of an
+ * array with each item being a diagram (a name and a content).
+ * @param {string} plugin - name of the plugin
+ * @param {string} description - description of the diagram
+ * @returns {Promise<object>} Promise with the diagram information on success
+ * otherwise an error.
+ */
+export async function generateDiagram(plugin, description) {
```
Comment (Zorin95670) [inline src/services/AIService.js:12]: A default timeout is configured to 15s, configure it for 15min or another increased time.

File: src/components/form/CreateAIModelForm.vue [hunk 2]
```diff
@@ -0,0 +1,231 @@
...
+    });
+}
+
+/**
+ * Create a new model folder and its parent folders if necessary.
+ * Emit a positive notification on success and redirect to model page.
+ * Otherwise, emit a negative notification.
+ * @returns {Promise<void>} Promise with nothing on success or error.
+ */
+async function onSubmit() {
```
Comment (Zorin95670) [inline src/components/form/CreateAIModelForm.vue:197]: you have to clean error too

File: src/components/card/DiagramsCard.vue [hunk 3]
```diff
@@ -51,6 +51,20 @@
               >
                 {{ $t('actions.models.create.button.template.name') }}
               </q-item>
+              <q-separator />
```
Comment (Zorin95670) [inline src/components/card/DiagramsCard.vue:54]: prefere to have:
```html
<template v-if="HAS_BACKEND">
  <q-separator />
  <q-item ...
 </template>
```

File: changelog.md [hunk 4]
```diff
@@ -26,6 +26,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 * Improve dockerfile with version of plugins as argument. 
 * Export diagram as svg.
 * Error management on monaco editor and error footer.
+* Generate diagrams from AI proxy. The proxy is actually called by the Leto-Modelizer-Api.
```
Comment (Zorin95670) [inline changelog.md:29]: remove  The proxy is actually called by the Leto-Modelizer-Api.

**Review by aboubakar14 — APPROVED** [review]

**Pull request merged**